재호/1935.py

```diff
@@ -0,0 +1,37 @@
+def is_expression(char):
+    return char == '*' or char == '+' or char == '/' or char == '-'
+
+
+def cal(a, b, exp):
+    if exp == '*':
+        return a * b
+    if exp == '/':
+        return a / b
+    if exp == '+':
+        return a + b
+    if exp == '-':
+        return a - b
+
+
+def solution():
+    char_dict = {}
+    N = int(input())
+    exps = input()
+    for i in range(N):
+        char_dict[chr(i + 65)] = int(input())
+
+    num_stack = []
+    for exp in exps:
+        if exp in char_dict.keys():
+            exp = char_dict[exp]
+        if is_expression(exp):
+            last_num = num_stack.pop()
+            last_second_num = num_stack.pop()
+            num_stack.append(cal(last_second_num, last_num, exp))
+        else:
+            num_stack.append(exp)
```

Comment:
```suggestion
    for exp in exps:
        if exp in char_dict.keys():
            num_stack.append(char_dict[exp])
        else:
            last_num = num_stack.pop()
            last_second_num = num_stack.pop()
            num_stack.append(cal(last_second_num, last_num, exp))
```
후위표기식에서 알파벳이 아니라면 무조건 위 4개의 연산자이기 때문에 `is_expression()`이라는 함수가 없어도 무관할거 같아요! 
그리고 for문 안에서 알파벳/연산자 구분만 필요하기 때문에 1번째 if문과 마지막 else문을 합치는 것도 가능해보입니다.

Reply:
아 맞네요! 서영님 말씀처럼 줄일 수 있을 것 같습니다!
감사합니다!